Autopilot/Inc/TelemDriver.hpp

```diff
@@ -0,0 +1,48 @@
+#ifndef TELEM_DRIVER_HPP
+#define TELEM_DRIVER_HPP
+
+
+#include <ctime>
+#include <cstdint>
+
+struct TelemToPMData {
```

Comment:
What you've got here is what we've called the PIGO and POGI structs (plane in ground out, plane out ground in). They probably shouldn't be defined in here though because a few different people will need to make use of them. Coordinate with Jingting (working on the mavlink encoder/decoder that you'l be using) and Laye/Avani(working on the path manager state machine) to figure out where everyone wants this data.

Reply:
Hi, I thought as much, so I implemented a header with both PIGO and POGI structs called TelemPathManagerInterface I believe that is also in this PR. I put it here as well because of something I read while trying to understand the Singleton style.

Reply:
The interface file will be in the autopilot/Inc directory too, so everyone can access it.